### ISSUE TITLE
Rewrite internal redirects in output

### ISSUE THREAD

**Issue by mlbrgl** [owid/owid-grapher#1148]
## Core problem

We need to keep redirects in place to not break incoming links from external sites. However there is no reason to keep outdated and redirected URLs in our content. As a matter of fact, the SEO audit mentioned it as an issue with regards to our crawl budget (see https://www.notion.so/owid/Update-internal-links-that-result-in-a-redirect-dc38b99a65aa4b969c6240a0a2698023) 

## Proposed solution
Initiated in #1147, the internal URL resolver could be applied on all links (not just prominent links) to fix the issue on the output.

## Alternatives
Additionally, we could surface those in a content report for manually fixing by authors. 
Ultimately, we could also provide tools to batch fix them or even rewrite content automatically when we're confident in the system's capabilities.

**Comment by mlbrgl** [comment]
We should wait at least a few weeks after #1147 went live (16 Feb) and enabled the resolver for prominent links to surface any bugs before launching at scale.

**Comment by larsyencken** [comment]
We triaged it as `medium` priority rather than `low`, because it came up as something we should fix in an earlier SEO audit.

**Comment by larsyencken** [comment]
Reviewed now with @danyx23, seems lower prio than before. Closing as `wontfix`, although if we have the opportunity to solve it as we move to a new rendering pipeline, we should.

**Comment by marcelgerber** [comment]
Just some additional metadata: We now have some information around this in Datasette, and found out that we currently have 216 links and embeds that point to a redirect rather than the current chart id: https://owid-datasette-y43qr.ondigitalocean.app/owid/post_charts?through_redirect__exact=1
